gh-codeql

```diff
@@ -14,8 +14,9 @@ error() {
 rootdir="$(dirname "$0")"
 channel="$(gh config get extensions.codeql.channel 2> /dev/null)" || : # Suppress an error and return empty if the field doesn't exist
 version="$(gh config get extensions.codeql.version 2> /dev/null)" || : # Suppress an error and return empty if the field doesn't exist
+local_pinning="$(gh config get extensions.codeql.local-pinning 2> /dev/null)" || : # Suppress an error and return empty if the field doesn't exist
 
-if [ -e .codeql-version ]; then
+if [ "$local_pinning" = "true" -a -e .codeql-version ]; then
```

Comment:
minor: If a `.codeql-version` is ignored because local_pinning is disabled, I think there should be a warning.

Reply:
I've changed how we handle local version support and added a warning when a local version is specified, but local version support is disabled to notify a user.